docs/progress.md

```
@@ -19,11 +19,11 @@
 
 ## Persoonlijke Workshop Informatie
 
-**Deelnemer Naam:** `[Jouw Volledige Naam]`  
-**Workshop Datum:** `[Voeg vandaag's datum toe]`  
-**Doel:** `[Wat wil je bereiken? bijv., "Mijn eerste CI/CD pipeline bouwen"]`  
-**GitHub Gebruikersnaam:** `[jouw-github-gebruikersnaam]`  
-**Verwachte Site URL:** `https://[jouw-gebruikersnaam].github.io/devops-workshop-vitepress/`
+**Deelnemer Naam:** `Elijah Hubrechts`  
+**Workshop Datum:** `27/11/25`  
+**Doel:** ` Mijn eerste CI/CD pipeline bouwen`  
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)